week-3/implement/is-proper-fraction.js

```diff
@@ -33,3 +33,11 @@
 // Explanation: The fraction 3/3 is not a proper fraction because the numerator is equal to the denominator. The function should return false.
 
 // These acceptance criteria cover a range of scenarios to ensure that the isProperFraction function handles both proper and improper fractions correctly and handles potential errors such as a zero denominator.
+
+function isProperFraction(Numerator,Denominator) {
```

Comment:
you didnt check if the denominator is 0. your variable names should start with a small letter. i.e `numerator`